README.md

```diff
@@ -13,5 +13,24 @@ npm install
 npm start
 ```
 
-Albert will also be available on mobile devices as an "installable" progressive web
-application via GitHub Pages once this repository has been made publicly available.
+## e2e Testing
+
+- Start up local many-server. ex: run `./scripts/run.sh` in many-framework
+- Make a duplicate of `e2e/.env.example` and name it `.env` in `e2e/`. Result
+  should be `e2e/.env`
+- Edit the variables in `e2e/.env`
+  - `MAIN_WALLET_PEM` should be the PEM file of the address with initial tokens
+    found in `ledger_state.json5` in many-framework.
+  ```
+  initial: {
+      "maffbahksdwaqeenayy2gxke32hgb7aq4ao4wt745lsfs6wijp": {
+          "MFX": 1000000000
+      }
+  },
+  ```
+  - Add the three seed phrses
```

Comment:
`phrases`.

We already have some testing accounts in the genesis file, can we use those?

Same with our dev `id1.pem` file, could we provide it to users/dev?

I think users and devs should be able to build/run our tests without intervention.

This could be another PR, however.

WDYT?

Reply:
default values for pem and seed phrases have been added.

Reply:
@hansl Pinging you here following our 1:1.